streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -102,6 +114,8 @@
      * This is equivalent to calling {@link #to(String)} and {@link org.apache.kafka.streams.kstream.KStreamBuilder#table(String)}.
      *
      * @param topic         the topic name
+     *
+     * @return a {@link KTable} that contains the exact same records as this {@link KTable}
```

Comment:
Nit: how about "a new {@link KTable} ... as this {@link KTable}"?

Ditto for all `through` calls of KTable and KStream